macros/macros.in

```diff
@@ -234,14 +234,14 @@ package or when debugging this package.\
 %_exec_prefix		%{_prefix}
 %_bindir		%{_exec_prefix}/bin
 %_sbindir		%{_exec_prefix}/sbin
-%_libexecdir		%{_exec_prefix}/libexec
+%_libexecdir		%{_exec_prefix}/lib
 %_datadir		%{_prefix}/share
 %_docdir		%{_datadir}/doc
 %_sysconfdir		/etc
 %_sharedstatedir	%{_prefix}/com
 %_localstatedir		/var
 %_lib			lib
-%_libdir		%{_exec_prefix}/%{_lib}
+%_libdir		%{_exec_prefix}/%{_lib}/%({__dpkg_architecture} -qDEB_HOST_MULTIARCH)
```

Comment:
If debbuild intends to target Debian systems, it should follow the filesystem hierarchy that Debian systems have.

It's aggravating that Debian doesn't follow [the standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.pdf), but it's important that things install into the right place.

So sadly, yes, debbuild needs to deviate from RPM in this regard. Actually, it would probably make sense for RPM to support these paths on Debian systems, too.